YouVsVirus/Assets/Scripts/Components/CreatePopulation.cs

```diff
@@ -123,7 +127,7 @@ private float GetCellRadius()
         }
 
         /// <summary>
-        /// Calucaltes the number of rows and columns of a grid of quadratic cells of given cell sidelength on the map.
+        /// Calclates the number of rows and columns of a grid of quadratic cells of given cell sidelength on the map.
```

Comment:
@maccxs  You fixed my spelling error with a spelling error of your own ;)

Reply:
That is my super power 👍